src/search_engines.js

```diff
@@ -444,6 +444,79 @@ class GoogleSearch {
     }
     return false;
   }
+
+  changeImageSize(size) {
```

Comment:
I think it's better to use as input to the function something more descriptive than `'i', 'l', 'e'`. Javascript doesn't have enums but you can simulate them: https://stackoverflow.com/q/287903/1014208